CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/Modules")
 ################ PROJECT VERSION ####################
 set(PROJECT_VERSION_FULL "0.2.0-dev1")
 set(PROJECT_SO_VERSION 7)
-set(HEX_VERSION_OVERRIDE "0x200") # For CMake < 3.13
+set(HEX_VERSION_OVERRIDE "0x109") # For CMake < 3.13
```

Comment:
Accidental reversion in the last commit.

Reply:
Noooo. It's one of those days, lol